is-provisioning-connector-archetype/src/main/resources/archetype-resources/src/main/java/__connector_name__ProvisioningConnector.java

```diff
@@ -0,0 +1,62 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package ${package};
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.common.model.Property;
+import org.wso2.carbon.identity.provisioning.AbstractOutboundProvisioningConnector;
+import org.wso2.carbon.identity.provisioning.IdentityProvisioningConstants;
+import org.wso2.carbon.identity.provisioning.IdentityProvisioningException;
+import org.wso2.carbon.identity.provisioning.ProvisionedIdentifier;
+import org.wso2.carbon.identity.provisioning.ProvisioningEntity;
+
+import java.util.Properties;
+
+/**
+ * Outbound Provisioning Connector for ${connector_name}
+ */
+public class ${connector_name}ProvisioningConnector extends AbstractOutboundProvisioningConnector {
+
+    private static final Log log = LogFactory.getLog(${connector_name}ProvisioningConnector.class);
+    private ${connector_name}ProvisioningConnectorConfig configHolder;
+
+    @Override
```

Comment:
Add method comments

Reply:
Please check the other places as well

Reply:
Since this is override method, is it necessary to add method level comments?

Reply:
without those comments, users may not have an idea about what this method will do. They don't have to override all the methods. Thus these comments may help them to choose the one they want.

Reply:
Fixed it in https://github.com/wso2-extensions/archetype-is/pull/2/commits/f27bff36ad98d3021597adecae3f16f4ef820c85